lib/core/src/Cardano/Wallet/Primitive/AddressDerivation/Shelley.hs

```diff
@@ -305,20 +305,20 @@ instance DelegationAddress ('Testnet pm) ShelleyKey where
     delegationAddress paymentK stakingK =
         Address $ BL.toStrict $ runPut $ do
             putWord8 (base + networkId)
-            putByteString . blake2b224 $ paymentK
-            putByteString . blake2b224 $ stakingK
+            putByteString . blake2b256 $ paymentK
+            putByteString . blake2b256 $ stakingK
       where
         base = 0
-        networkId = 0
+        networkId = 1 -- NOTE: Should be 0. Hack to support FF.
```

Comment:
site-note: this should be `FIXME`, not `NOTE`. FIXME is used for code that will require change otherwise the software won't work. NOTE is for giving precision. (and TODO is more of a nice-to-have, but the software will work without).